src/test/java/SmokeTest.java

```diff
@@ -0,0 +1,87 @@
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import pages.*;
+
+import java.io.IOException;
+
+public class SmokeTest extends BaseTest{
+
+    private String emailAddress = randomEmail();
+
+    private String password = randomPassword();
+    private String actualProductName;
+    private double actualPrice;
+    private int orderId;
+    private String productName = "blue jacket";
+    private String thankYouText = "Thank you for your purchase!";
+    private String accountIsCreated = "Thank you for registering with Fake Online Clothing Store.";
```

Comment:
String generators, Strings and any other form of hardcoding of said values should be placed in a static class either called "String resources" or "Utilities"